apps/balancer-tools/src/app/poolsimulator/(components)/Menu.tsx

```diff
@@ -231,7 +242,7 @@ export default function Menu() {
     ) {
       return <Spinner />;
     }
-    return <div className="text-slate12">Analysis Menu</div>;
+    return <AnalysisPoolParamsForm />;
```

Comment:
Should we make the same pattern and let both `AnalysisPoolParamsForm` and `IndexMenu` components in or outside the `Menu` file?

Reply:
I think we can discuss this better on https://linear.app/bleu-llc/issue/BAL-548/enable-analysis-page-with-dynamic-data but I would do both outside of `Menu`